containers/sarek/Dockerfile

```diff
@@ -1,18 +1,10 @@
-FROM debian:8.9
+FROM nfcore/base:latest
 
 LABEL \
-  author="Maxime Garcia" \
+  authors="Maxime.Garcia@scilifelab.se, Szilveszter.Juhos@scilifelab.se" \
   description="Image with tools used in Sarek" \
-  maintainer="maxime.garcia@scilifelab.se"
+	maintainer="Maxime Garcia <maxime.garcia@scilifelab.se>, Szilveszter Juhos <Szilveszter.Juhos@scilifelab.se>"
 
-# Export PATH
-ENV \
-  MANTA_INSTALL_PATH=/opt/manta \
-  PATH=/opt/bcftools/bin:/opt/htslib/bin:/opt/samtools/bin:$PATH \
-  STRELKA_INSTALL_PATH=/opt/strelka
-
-# Build container
-ADD \
-  build.sh /usr/bin/
-RUN \
-  build.sh
+COPY environment.yml /
+RUN conda env update -n root -f /environment.yml && conda clean -a
+ENV PATH /opt/conda/bin:$PATH
```

Comment:
we can get rid of this last line:
https://gitter.im/nf-core/Lobby?at=5b59f41bd2f0934551d30d5d

Reply:
tried, and was not working as expected, so I prefer to leave it as it is now, and improve when needed

Reply:
we will keep now with the ENV